tests/routes/pro/get_offerers_names_test.py

```diff
@@ -7,58 +7,166 @@
 from tests.conftest import TestClient
 
 
-class Returns200:
+class Returns200ForProUser:
+    def _setup_offerers_for_pro_user(self, user):
+        user_offerer = offers_factories.OffererFactory()
```

Comment:
```suggestion
        owned_offerer = offers_factories.OffererFactory()
```

Parce que sinon il y a confusion avec `user_offerer` qui est un vrai objet